freqtrade/strategy/interface.py

```diff
@@ -852,6 +856,29 @@ def min_roi_reached(self, trade: Trade, current_profit: float, current_time: dat
         else:
             return current_profit > roi
 
+    def ft_check_timed_out(self, side: str, trade: Trade, order: Dict,
+                           current_time: datetime) -> bool:
+        """
+        FT Internal method.
+        Check if timeout is active, and if the order is still open and timed out
+        """
+        timeout = self.config.get('unfilledtimeout', {}).get(side)
+        ordertime = arrow.get(order['datetime']).datetime
+        if timeout is not None:
+            timeout_unit = self.config.get('unfilledtimeout', {}).get('unit', 'minutes')
+            timeout_kwargs = {timeout_unit: -timeout}
+            timeout_threshold = current_time + timedelta(**timeout_kwargs)
+            timedout = (order['status'] == 'open' and order['side'] == side
+                        and ordertime < timeout_threshold)
+            if timedout:
+                return True
+        time_method = 'check_sell_timeout' if order['side'] == 'sell' else 'check_buy_timeout'
```

Comment:
could just as well use `self.check_sell_timeout` and `self.check_buy_timeout` instead of strings.